src/libs/ajax/leonardo/LeoCookiesDataClient.ts

```diff
@@ -0,0 +1,41 @@
+import { AbortOption, FetchFn } from '@terra-ui-packages/data-client-core';
```

Comment:
Consider adding this file to the leo data client package along with the disk client. It has no dependencies in terra UI proper as the authed function is injected. 

Really shows you did a good job of splitting up dependencies that this new code can trivially be packaged!